### PR TITLE
Add start options for merge and force

### PR DESCRIPTION
**Pull request by jvz**
This exposes the merge mappings and force creation settings into a new
ElasticsearchBeyonder.start() method along with adding default values to
the Defaults class.

Note that this idea came to me and @divideby0 while developing a benchmarking application. We generally like to clear out indices in between runs (especially when switching between scenarios which may have completely different mappings, settings, etc.). I noticed you left a TODO comment here, so I've gone ahead and implemented it on the `RestClient` version as the other version is marked for removal eventually.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@ when an elasticsearch client starts.
         <elasticsearch.version>6.1.3</elasticsearch.version>
         <test.elasticsearch.version>${elasticsearch.version}</test.elasticsearch.version>
         <slf4j.version>1.7.13</slf4j.version>
-        <log4j.version>2.8.1</log4j.version>
+        <log4j.version>2.10.0</log4j.version>
```
Comment (dadoonet) [inline pom.xml]: Could you make that change in another PR?

**Comment by jvz** [comment]
I repushed this with a commit signoff in lieu of any CLAs or whatever. I hope that's good enough!

File: pom.xml (hunk 1 above)
Comment (jvz) [reply pom.xml]: Yeah. Would you like any other version updates addressed there? I mainly knew about that one since I'm one of their maintainers.

**Comment by jvz** [comment]
Alright, I'll push up a changed commit without the dependency upgrade. Will include a README update along with tests. Thanks for the quick feedback!

File: pom.xml (hunk 1 above)
Comment (dadoonet) [reply pom.xml]: I generally prefer updating librairies one by one (one per PR) so it's easier to track the changes even if it's super small.

**Comment by dadoonet** [comment]
While we are at it I think you could add in your commit message:

```
Closes #8.
```

At it seems that this will close #8

**Comment by jvz** [comment]
I think this addresses everything.

**Comment by jvz** [comment]
Thanks again for the quick feedback!

File: src/test/java/fr/pilato/elasticsearch/tools/BeyonderRestIT.java [hunk 2]
```diff
@@ -152,4 +156,29 @@ public void testAliases() throws Exception {
         Map<String, Object> response = asMap(client.performRequest("GET", "/_alias/foo"));
         assertThat(response, hasKey("test_aliases"));
     }
+
+    @Test
+    public void testMergeDisabled() throws Exception {
+        ElasticsearchBeyonder.start(client, "elasticsearch");
```
Comment (dadoonet) [inline src/test/java/fr/pilato/elasticsearch/tools/BeyonderRestIT.java]: According to the fact that the default path name is `elasticsearch`, I think we can call here instead:

```java
ElasticsearchBeyonder.start(client);
```

Not important but that will at least tell my IDE to stop proposing making `start(RestClient)` as a private method ;)

File: src/test/java/fr/pilato/elasticsearch/tools/BeyonderRestIT.java [hunk 3]
```diff
@@ -152,4 +156,29 @@ public void testAliases() throws Exception {
...
+        String expectedMapping = getMapping("twitter");
+        ElasticsearchBeyonder.start(client, "models/mergedisabled", false, false);
+        String actualMapping = getMapping("twitter");
+        assertThat(actualMapping, is(expectedMapping));
+    }
+
+    @Test
+    public void testForceEnabled() throws Exception {
+        ElasticsearchBeyonder.start(client, "elasticsearch");
```
Comment (dadoonet) [inline src/test/java/fr/pilato/elasticsearch/tools/BeyonderRestIT.java]: Same as previous comment

File: src/test/java/fr/pilato/elasticsearch/tools/BeyonderRestIT.java (hunk 2 above)
Comment (jvz) [reply src/test/java/fr/pilato/elasticsearch/tools/BeyonderRestIT.java]: Oh good point! I, too, get annoyed by that IDE hint, though I suppose it's just trying to nudge us into the TDD direction? :)

**Comment by jvz** [comment]
I've pushed those changes you requested.
